### DISCUSSION TITLE
I want to start a new resource in a project that already has admin panel but i got an error

### DISCUSSION THREAD

**Discussion by pamellaneco** [filamentphp/filament#11231] [Help] [answered]
### Package

Panel builder

### Package Version

v3.0.0

### How can we help you?

Hi, I want to start a new resource in my project (that already has a admin panel) using laravel10, sail and filament and I ran this command line: 
`sail artisan make:filament-resource Clients --simple --generate`

and got this error: 
![image](https://github.com/filamentphp/filament/assets/104801541/4c41225c-540b-45ee-90dc-95b37cb26527)

do you know what it can be?
@lea

**Comment by leandrocfe** [comment]
Do you have Models\Clients.php class in your project?
If you are following the Laravel convention, the model should be singular (Client), not plural (Clients).

**Reply by pamellaneco** [reply]
I deleted the resource in order to create a new one following the Laravel convention "Client".
But about the model, I have to create manually or there is some command to add this in a automatic way?

`sail artisan make:filament-resource Client --simple --generate`

**Reply by leandrocfe** [reply, marked as answer]
`php artisan make:model Client` ?

**Reply by pamellaneco** [reply]
@leandrocfe I created the model and migration but it throws an error when I try to migrate the table:

this is the error:
![image](https://github.com/filamentphp/filament/assets/104801541/2ce0e06d-d1e7-4d65-b117-abb8fe2caeb7)

this is the code in migration file:
``` 
<?php


use Illuminate\Database\Migrations\Migration;
use Illuminate\Database\Schema\Blueprint;
use Illuminate\Support\Facades\Schema;

return new class extends Migration
{
    /**
     * Run the migrations.
     */
    public function up(): void
    {
        Schema::create('clients', function (Blueprint $table) {
            $table->id();
            $table->string('company_name');
            $table->int('cnpj'); 
            $table->string('email'); 
            $table->int('phone');
            $table->text('description')->nullable();
            $table->timestamps();
        });
    }

    /**
     * Reverse the migrations.
     */
    public function down(): void
    {
        Schema::dropIfExists('clients');
    }
};

**Reply by leandrocfe** [reply]
It isn't a Filament issue, you have a database connection error or something like this.
If you are using sail, you should run `sail artisan migrate`

**Reply by pamellaneco** [reply]
okay, I'll try to figure out whats going on. I'm already running `sail artisan migrate`

edit: if you would like to know, It was because I used a wrong data type "int" instead of "integer"
